### PR TITLE
Update deep link with new URLs

### PR DESCRIPTION
**Pull request by whydna**
Updates the deeplinking documentation for the added routes.

### CODE REVIEW

File: TMTumblrSDK/AppClient/TMTumblrAppClient.h [hunk 1]
```diff
@@ -21,10 +21,16 @@
...
++ (void)viewExplore;
+
+// View the activity page for a given blog. If blogName is null, will display the primary blog.
++ (void)viewActivity:(NSString *)blogName;
+
 /// View a tag
 + (void)viewTag:(NSString *)tag;
 
-/// View a blog
+/// View a blog. If blogName is null, will display the primary blog
```
Comment (paulrehkugler) [inline TMTumblrSDK/AppClient/TMTumblrAppClient.h]: `nil` not `null`

File: TMTumblrSDK/AppClient/TMTumblrAppClient.h [hunk 2]
```diff
@@ -21,10 +21,16 @@
 /// View the authenticated user's dashboard
 + (void)viewDashboard;
 
+/// View the explore page
++ (void)viewExplore;
+
+// View the activity page for a given blog. If blogName is null, will display the primary blog.
```
Comment (paulrehkugler) [inline TMTumblrSDK/AppClient/TMTumblrAppClient.h]: `nil` not `null`

**Comment by irace** [comment]
@whydna This is good to go?

**Comment by whydna** [comment]
@irace  Yep!

File: TMTumblrSDK/AppClient/TMTumblrAppClient.h [hunk 3]
```diff
@@ -21,10 +21,16 @@
...
+
+// View the activity page for a given blog. If blogName is nil, will display the primary blog.
++ (void)viewActivity:(NSString *)blogName;
+
 /// View a tag
 + (void)viewTag:(NSString *)tag;
 
-/// View a blog
+/// View a blog. If blogName is nil, will display the primary blog
 + (void)viewBlog:(NSString *)blogName;
```
Comment (irace) [inline TMTumblrSDK/AppClient/TMTumblrAppClient.h]: @whydna Think we should add a `viewPrimaryBlog` method that just calls through with `nil`?

Comment (whydna) [inline TMTumblrSDK/AppClient/TMTumblrAppClient.h]: Yeah, makes sense. Will do the same for `viewActivity`?

Comment (mattbischoff) [inline TMTumblrSDK/AppClient/TMTumblrAppClient.h]: Yup!

**Pull request merged**